tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -81,6 +82,7 @@ sub run {
     die "HA cluster needs at least 2 nodes. Check 'NODE_COUNT' parameter." if ($ha_enabled && (get_var('NODE_COUNT') <= 1));
 
     set_var('FENCING_MECHANISM', 'native') unless ($ha_enabled);
+    set_var('ISCSI_ENABLED', (get_var('FENCING_MECHANISM') eq 'sbd') ? 'true' : 'false');
```

Comment:
There is `check_var` API call if you wanna simplify it a little. 
```
set_var('ISCSI_ENABLED', (check_var('FENCING_MECHANISM',  'sbd')  ?  'true' : 'false');
```

Does not do much difference though.